src/EasyAbp.Abp.SettingUi.Web/Pages/SettingManagement/SettingUi/Index.js

```diff
@@ -46,23 +46,35 @@
         })
     }
 
-    $('#tabs-nav .nav-item .nav-link').click(function () {
-        var _this = $(this);
-        if (_this.attr('data-bs-target') !== undefined) {
-            return;
-        }
+    var bind_nav_event_fn = function (element) {
         var btn_set = setInterval(function () {
-            var _id = '#' + _this.data('id') + " ";
+            var _id = '#' + element.data('id') + " ";
```

Comment:
There's a legacy here, when we click **FeatureManagement Tab** `$(_id + " form").length` It's always 0，
So the value of id should be`var _id = '#' + element.data('id').replace(/\./g, '-') + " ";`Then Line10~Line12 can also be removed

Reply:
[AbpSettingManagement index.js ](https://github.com/abpframework/abp/blob/c9e0df1884b0d5cc9ce7aa5822b0c4c34f3c6939/modules/setting-management/src/Volo.Abp.SettingManagement.Web/Pages/SettingManagement/Index.js#L11C40-L11C40)